classic/FindleyLake/parameters/global_param.FindleyLake.txt

```diff
@@ -0,0 +1,102 @@
+MODEL_STEPS_PER_DAY   24
+SNOW_STEPS_PER_DAY	  24
+RUNOFF_STEPS_PER_DAY  24
+
+STARTYEAR	1970
+STARTMONTH	1
+STARTDAY	1
+ENDYEAR 	1979
+ENDMONTH	12
+ENDDAY		31
+CALENDAR    PROLEPTIC_GREGORIAN
+
+NODES                 3      # number of soil thermal nodes
+FULL_ENERGY TRUE
+FROZEN_SOIL	FALSE
+AERO_RESIST_CANSNOW AR_406
+
+FORCING1      {$VIC_SAMPLE_DATA}/classic/FindleyLake/forcings/livneh_NAmerExt_15Oct2014.FindleyLake_
+FORCE_FORMAT ASCII
+FORCE_TYPE    PREC 	   prec
+FORCE_TYPE    AIR_TEMP 	   temp
+FORCE_TYPE    SWDOWN       shortwave
+FORCE_TYPE    LWDOWN 	   longwave
+FORCE_TYPE    PRESSURE 	   air_pressure
+FORCE_TYPE    VP           vapor_pressure
+FORCE_TYPE    WIND         wind
+FORCE_STEPS_PER_DAY  24
+FORCEYEAR  1970
+FORCEMONTH 1
+FORCEDAY   1
+GRID_DECIMAL 4
+WIND_H        10.0
+
+SOIL            {$VIC_SAMPLE_DATA}/classic/FindleyLake/params/soil_param.FindleyLake
+VEGLIB          {$VIC_SAMPLE_DATA}/classic/FindleyLake/params/veg_lib_IGBP
+VEGPARAM        {$VIC_SAMPLE_DATA}/classic/FindleyLake/params/veg_param.FindleyLake
+ROOT_ZONES      3
+VEGLIB_FCAN     FALSE
+VEGPARAM_LAI    TRUE
+VEGPARAM_FCAN   TRUE
+VEGPARAM_ALB    TRUE
+LAI_SRC         FROM_VEGPARAM
+FCAN_SRC        FROM_VEGPARAM
+ALB_SRC         FROM_VEGPARAM
+LAKES           {$VIC_SAMPLE_DATA}/classic/FindleyLake/params/lake_param.FindleyLake
+LAKE_PROFILE    TRUE
+EQUAL_AREA      TRUE
+RESOLUTION      1.29
+SNOW_BAND       1
+
+#INIT_STATE  {$VIC_SAMPLE_DATA}/classic/FindleyLake/state/state.FindleyLake.19800101
+STATENAME   {$VIC_SAMPLE_DATA}/classic/FindleyLake/state/state.FindleyLake
+STATEYEAR   1980
+STATEMONTH  1
+STATEDAY    1
+
+RESULT_DIR  {$VIC_SAMPLE_DATA}/classic/FindleyLake/results
```

Comment:
For the base directory for the Stehekin, we had `${VIC_SAMPLE_RESULTS}` - do we want to keep it this way or is there a good reason for changing it?

Reply:
Done.